packages/back-end/src/controllers/features.ts

```diff
@@ -513,6 +518,71 @@ export async function postFeatureRebase(
     status: 200,
   });
 }
+export async function postFeatureRequestReview(
+  req: AuthRequest<
+    {
+      comment: string;
+    },
+    { id: string; version: string }
+  >,
+  res: Response
+) {
+  const { org } = getContextFromReq(req);
+  const { id, version } = req.params;
+  const { comment } = req.body;
+  const feature = await getFeature(org.id, id);
+  if (!feature) {
+    throw new Error("Could not find feature");
+  }
+
+  const revision = await getRevision(org.id, feature.id, parseInt(version));
+  if (!revision) {
+    throw new Error("Could not find feature revision");
+  }
+  if (revision.status !== "draft") {
+    throw new Error("Can only request review if is a draft");
+  }
+  await markRevisionAsReviewRequested(revision, res.locals.eventAudit, comment);
+  res.status(200).json({
+    status: 200,
+  });
+}
+
+export async function postFeatureReviewOrComment(
+  req: AuthRequest<
+    {
+      comment: string;
+      review: ReviewSubmittedType;
+    },
+    { id: string; version: string }
+  >,
+  res: Response
+) {
+  const { org } = getContextFromReq(req);
+  const { id, version } = req.params;
+  const { comment, review } = req.body;
+  const feature = await getFeature(org.id, id);
+  if (!feature) {
+    throw new Error("Could not find feature");
+  }
+
+  const revision = await getRevision(org.id, feature.id, parseInt(version));
+  if (!revision) {
+    throw new Error("Could not find feature revision");
+  }
+  if (revision.status === "draft") {
```

Comment:
Aren't there other statuses like `published` or `discarded` that we also want to block here?   Might be easier to whitelist the statuses we do allow instead.